definitions/private-law/json/CaseEventToFields/OtherPeopleInTheCase/CaseEventToFields.json

```diff
@@ -7,7 +7,7 @@
     "DisplayContext": "COMPLEX",
     "PageID": 1,
     "PageDisplayOrder": 1,
-    "PageFieldDisplayOrder": 1,
+    "PageFieldDisplayOrder": 2,
```

Comment:
Do we need this change in display order?

Reply:
there were field added as part of PRL-660, so added that again